docs/management/admin/response-actions-history.asciidoc

```diff
@@ -3,6 +3,12 @@
 
 {elastic-defend} keeps a log of the <<response-actions,response actions>> performed on endpoints, such as isolating a host or terminating a process. The log displays when each command was performed, the host on which the action was performed, the {kib} user who requested the action, any comments added to the action, and the action's current status.
 
+.Requirement
+[sidebar]
+--
+You must have the *Actions Log Management* <<endpoint-management-req,user privilege>> to access this feature.
```

Comment:
```suggestion
You must have the *Actions Log Management* <<endpoint-management-req,privilege>> to access this feature.
```
I noticed that near the top of this PR you referred to "Kibana privileges", so I would suggest we stay consistent in how we refer to privs. Not sure if "Kibana privilege" or just "privilege" would be better here.

Reply:
"User privilege" also appears in this PR in `/response-actions.asciidoc`, so I'm not sure which version we should go with